sangchun-html/mypage.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/styles.css" />
+    <link rel="stylesheet" href="/css/sangchun/mypage.css" />
+    <title>Mypage</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="mypage-container">
+        <div class="nav-top">
+          <i class="fa-solid fa-chevron-left"></i>
```

Comment:
```suggestion
          <a href="#"><i class="fa-solid fa-chevron-left"></i></a>
```
뒤로가기 버튼 a 태그로 감싸서 링크 걸어주면 좋을 것 같아용
링크는 임시로 "#"으로~~ 
나중에 해당하는 html 파일 이름 넣읍시당